### PR TITLE
Add processor for border-radius

### LINKED ISSUE #5: Processor for adding border radius
## Description

There are some elements that need some border-radius, and they should be configurable form the WP editor. Most commonly used are 8px and 12px. 

Gutenberg blocks don't support this by default, so we are planning on using classes like these: `has-border-radius-8px` or `has-border-radius-12px`.

It could be used inside any tag.

## Style
We just have to add the correct border-radius to the element.

## Where to check
Inside `/common-styles/` page -> Divs with custom border-radius

### PR DESCRIPTION
**Pull request by iamuchejude**
Fixes #5

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 6c0678d39936c0f6d8cee84e066f42663b05f6ed

https://frontity-org-gwa8t5af1.now.sh

**Comment by iamuchejude** [comment]
@DAreRodz I'm awaiting for your review.

File: packages/frontity-org-theme/src/processors/border-radius.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          ${node.props.css}
+          ${style}
+        `
+      }
+    })
+  }));
+};
+
+export default createBorderRadiusProcessors({
+  "has-border-radius-8px": css`
```
Comment (DAreRodz) [inline packages/frontity-org-theme/src/processors/border-radius.ts]: Hey, maybe we can change this to get the number of pixels from the class name? I guess that's the idea so we could change the number of pixels in the future just changing that value in the class.

Comment (iamuchejude) [reply packages/frontity-org-theme/src/processors/border-radius.ts]: Thanks for the feedback @DAreRodz.

The issue with this is that we might complicate the test function for the processor. For your implementation to work, we'll need a global class `has-border-radius` for all elements that have border-radius and another class with the size of the border-radius.

Comment (iamuchejude) [reply packages/frontity-org-theme/src/processors/border-radius.ts]: @SantosGuillamot What do you think?

Comment (SantosGuillamot) [reply packages/frontity-org-theme/src/processors/border-radius.ts]: I think what David was proposing was to keep using the same class, `has-border-radius-XXpx` and get the pixels with a regexp. This way we will be using just one class but it'd be more flexible so we can use it for other `border-radius` like `has-border-radius-16px`.

Comment (iamuchejude) [reply packages/frontity-org-theme/src/processors/border-radius.ts]: So, we will have 2 classes for elements with border-radius. One will be `has-border-radius` and the other will be `has-border-radius-XXpx`.

I'm gonna go update the classes on the wp-admin dashboard.

**Comment by michalczaplinski** [comment]
Hey @iamuchejude can this be merged already? 🙂

**Pull request merged**